config/webhooks/webhooks.yaml

```diff
@@ -0,0 +1,91 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: keda-admission
+  namespace: keda
+  labels:
+    app: keda-admission-webhooks
+    app.kubernetes.io/name: admission-webhooks
+    app.kubernetes.io/version: latest
+    app.kubernetes.io/component: admission-webhooks
+    app.kubernetes.io/part-of: keda
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: keda-admission-webhooks
+  template:
+    metadata:
+      labels:
+        app: keda-admission-webhooks
+        name: keda-admission-webhooks
+      name: keda-admission-webhooks
+    spec:
+      securityContext:
+        runAsNonRoot: true
+      serviceAccountName: keda-operator
+      containers:
+        - name: keda-admission-webhooks
+          image: ghcr.io/kedacore/keda-admission-webhooks:latest
+          command:
+            - /keda-admission-webhooks
+          args:
+            - --zap-log-level=info
+            - --zap-encoder=console
+            - --zap-time-encoding=rfc3339
+          imagePullPolicy: Always
+          resources:
+            requests:
+              cpu: 100m
+              memory: 100Mi
+            limits:
+              cpu: 1000m
+              memory: 1000Mi
```

Comment:
Let's update our doc PR to include this on https://keda.sh/docs/2.9/operate/cluster/#cluster-capacity, but for 2.10

Reply:
sure, omw